bears/python/VultureBear.py

```diff
@@ -3,42 +3,26 @@
 from dependency_management.requirements.PipRequirement import PipRequirement
 from vulture import Vulture
 
-CONFIDENCE_MAP = {
-    'attribute': 70,
-    'class': 70,
-    'function': 70,
-    'import': 95,
-    'property': 70,
-    'variable': 70,
-}
-
 
 def _find_unused_code(filenames):
     """
     :param filenames: List of filenames to check.
     :return: Generator of Result objects.
     """
-
-    def file_lineno(item):
-        return (item.filename.lower(), item.lineno)
-
-    vulture = Vulture()
+    vulture = Vulture(verbose=True)
     vulture.scavenge(filenames)
-    for item in sorted(
-            vulture.unused_funcs + vulture.unused_imports +
-            vulture.unused_props + vulture.unused_vars +
-            vulture.unused_attrs, key=file_lineno):
-        message = 'Unused {0}: {1}'.format(item.typ, item)
+    for item in vulture.get_unused_code(sort_by_size=True):
```

Comment:
Is sorting helpful here? I would guess that coala has its own sorting. And if it doesn't, I think we should sort by filenames.